tutorials/spectral/plot_SpectralVariance.py

```diff
@@ -132,19 +135,19 @@
 # slices from the non-overlapping spectrogram and taking their average.
 #
 # The resample spectral coefficient of variation can be computed with
-# :func:`~neurodsp.spectral.variance.compute_scv_rs`.
+# :func:`~.compute_scv_rs`.
 #
 
 ###################################################################################################
 
 # Calculate SCV with the resampling method
-freqs, t_inds, scv_rs = spectral.compute_scv_rs(sig, fs, nperseg=fs, method='bootstrap',
+freqs, t_inds, scv_rs = compute_scv_rs(sig, fs, nperseg=fs, method='bootstrap',
```

Comment:
lines again. 

If you want to check across files, you can run a linter on tutorial files (ex: `pylint tutorials/spectral/*`). I usually pipe this to a file (`pylint tutorials/spectral/* -> lints.txt` or something). It does pick up quite a lot of nonsense (because these files aren't module files and it screams about constants), but it can be useful for a quick search for line spacing and things like that.

Reply:
Fixed. I have a linter plugin for Atom but I missed these spacing issues. I can run additional checks via command line before pushing.